service/src/main/resources/db/migration/V373__calendar_event_time_reservation.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE calendar_event_time
+(
+    id                uuid PRIMARY KEY                  DEFAULT ext.uuid_generate_v1mc(),
+    created           timestamp with time zone NOT NULL DEFAULT now(),
+    updated           timestamp with time zone NOT NULL DEFAULT now(),
+    modified_at       timestamp with time zone NOT NULL,
+    modified_by       uuid                     NOT NULL REFERENCES evaka_user (id),
+    calendar_event_id uuid                     NOT NULL REFERENCES calendar_event (id) ON DELETE CASCADE,
+    date              date                     NOT NULL,
+    start_time        time without time zone   NOT NULL,
+    end_time          time without time zone   NOT NULL,
+    child_id          uuid REFERENCES child (id)
```

Comment:
Should we limit on DB level that there can only be one unique `calendar_event_id` + `child_id` combination?

Reply:
No, there should be possibility to add multiple time reservations for child, e.g. one for each guardian.